projects/telosb/SConscript.env

```diff
@@ -24,7 +24,11 @@ MSPVERSION = "1611"
 if   buildEnv['toolchain']=='mspgcc':
    flags = []
    flags += ['-mmcu=msp430f{0}'.format(MSPVERSION)]
-   flags += ['-Os']
+   if buildEnv['debug']==0:

```

Comment:
The -O0 option is needed by Eclipse to debug with gdb when the mspgcc toolchain is used.